### PR TITLE
Added collection for multi attribute

### PR DESCRIPTION
**Pull request by ananjaykumar2**

### CODE REVIEW

**Comment by kailash** [comment]
ok to test

File: src/test/resources/IUDX-Resource-Server-Consumer-APIs-V4.0.postman_collection.json [hunk 1]
```diff
@@ -8959,6 +9033,77 @@
...
+									"pm.test(\"Check response header\", function () {",
+									"    pm.response.to.have.header(\"Content-Type\",\"application/json\");",
+									"});",
+									"",
+									"// Test the response",
+									"pm.test(\"Check response body\", function () {    ",
+									"    const body = pm.response.json();",
+									"    pm.expect(body).to.have.property(\"title\", \"Success\");",
+									"    const resultsjsonData = body.results[0];",
+									"    pm.expect(resultsjsonData).to.have.property(\"count\", 957);",
```
Comment (kailash) [inline src/test/resources/IUDX-Resource-Server-Consumer-APIs-V4.0.postman_collection.json:9059]: Please correct the postman assertion test, the API response for count will not have count as property 

```json
{
    "type": "urn:dx:rs:success",
    "title": "Success",
    "results": [
        {
            "totalHits": 19
        }
    ]
}

```

**Review by kailash — CHANGES_REQUESTED** [review]
resolve postman assertion test.

File: src/test/resources/IUDX-Resource-Server-Consumer-APIs-V4.0.postman_collection.json (hunk 1 above)
Comment (ananjaykumar2) [reply src/test/resources/IUDX-Resource-Server-Consumer-APIs-V4.0.postman_collection.json]: done

**Comment by kailash** [comment]
retest this please

**Comment by kailash** [comment]
retest this please

**Pull request merged**